netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -261,7 +263,7 @@ public void operationComplete(ChannelFuture future) throws Exception {
                   // The http2Stream will be null in case a stream buffered in the encoder
                   // was canceled via RST_STREAM.
                   if (http2Stream != null) {
-                    http2Stream.setProperty(NettyClientStream.class, stream);
+                    http2Stream.setProperty(streamKey, stream);
```

Comment:
I needed to use a newer Netty to benchmark with the framer flush fix I sent. Will adjust this PR as needed depending on what order those changes go in